ETC.tsv

```diff
@@ -13413,7 +13413,7 @@ ETC_20150717_013414	{memo X}will revive you in the dungeon entrance outside.
 ETC_20150717_013415	Be careful when you delve into a dungeon!
 ETC_20150717_013416	You can obtain Gems by hunting in dungeons.
 ETC_20150717_013417	{memo X}Watch out! Gems can be lost on death!
-ETC_20150717_013418	Worship the Goddess to receive beneficial effects.
+ETC_20150717_013418	$Pray in front of a Goddess Statue to receive beneficial effects.
```

Comment:
I see "worship" as something more perpetual, so I'd be for "offering a prayer" instead. I don't see a need to stick to one, though. In principle, they express the same thing, so less repetition is better.